### ISSUE TITLE
Docker build error

### ISSUE THREAD

**Issue by alxsimon** [BELKHIR/shinyvcfanalyser#5]
When trying to build the docker container with
`sudo docker build -t shinyvcfmultisampleanalyser .`

I have the following error:
```
Step 17/112 : RUN cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py && rm -f get-pop.py
 ---> Running in 4e5d92de71c1
--2021-02-01 11:58:29--  https://bootstrap.pypa.io/get-pip.py
Resolving bootstrap.pypa.io (bootstrap.pypa.io)... 151.101.120.175, 2a04:4e42:1d::175
Connecting to bootstrap.pypa.io (bootstrap.pypa.io)|151.101.120.175|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1929903 (1.8M) [text/x-python]
Saving to: ‘get-pip.py’

     0K .......... .......... .......... .......... ..........  2% 3.95M 0s
    50K .......... .......... .......... .......... ..........  5% 6.64M 0s
   100K .......... .......... .......... .......... ..........  7% 3.96M 0s
   150K .......... .......... .......... .......... .......... 10% 4.72M 0s
   200K .......... .......... .......... .......... .......... 13% 4.63M 0s
   250K .......... .......... .......... .......... .......... 15% 3.77M 0s
   300K .......... .......... .......... .......... .......... 18% 3.71M 0s
   350K .......... .......... .......... .......... .......... 21% 15.5M 0s
   400K .......... .......... .......... .......... .......... 23% 8.63M 0s
   450K .......... .......... .......... .......... .......... 26% 1.77M 0s
   500K .......... .......... .......... .......... .......... 29% 1.55M 0s
   550K .......... .......... .......... .......... .......... 31% 4.15M 0s
   600K .......... .......... .......... .......... .......... 34% 6.29M 0s
   650K .......... .......... .......... .......... .......... 37% 5.13M 0s
   700K .......... .......... .......... .......... .......... 39% 15.5M 0s
   750K .......... .......... .......... .......... .......... 42% 5.32M 0s
   800K .......... .......... .......... .......... .......... 45% 3.84M 0s
   850K .......... .......... .......... .......... .......... 47% 6.40M 0s
   900K .......... .......... .......... .......... .......... 50% 6.25M 0s
   950K .......... .......... .......... .......... .......... 53% 2.53M 0s
  1000K .......... .......... .......... .......... .......... 55% 8.96M 0s
  1050K .......... .......... .......... .......... .......... 58% 9.41M 0s
  1100K .......... .......... .......... .......... .......... 61% 6.02M 0s
  1150K .......... .......... .......... .......... .......... 63% 4.17M 0s
  1200K .......... .......... .......... .......... .......... 66% 5.99M 0s
  1250K .......... .......... .......... .......... .......... 68% 6.29M 0s
  1300K .......... .......... .......... .......... .......... 71% 2.39M 0s
  1350K .......... .......... .......... .......... .......... 74% 4.46M 0s
  1400K .......... .......... .......... .......... .......... 76% 4.27M 0s
  1450K .......... .......... .......... .......... .......... 79% 6.59M 0s
  1500K .......... .......... .......... .......... .......... 82% 4.24M 0s
  1550K .......... .......... .......... .......... .......... 84% 5.87M 0s
  1600K .......... .......... .......... .......... .......... 87% 4.25M 0s
  1650K .......... .......... .......... .......... .......... 90% 5.04M 0s
  1700K .......... .......... .......... .......... .......... 92% 5.68M 0s
  1750K .......... .......... .......... .......... .......... 95% 4.04M 0s
  1800K .......... .......... .......... .......... .......... 98% 12.5M 0s
  1850K .......... .......... .......... ....                 100% 6.66M=0.4s

2021-02-01 11:58:29 (4.57 MB/s) - ‘get-pip.py’ saved [1929903/1929903]

Traceback (most recent call last):
  File "get-pip.py", line 24244, in <module>
    main()
  File "get-pip.py", line 199, in main
    bootstrap(tmpdir=tmpdir)
  File "get-pip.py", line 82, in bootstrap
    from pip._internal.cli.main import main as pip_entry_point
  File "/tmp/tmpnnsHCv/pip.zip/pip/_internal/cli/main.py", line 60
    sys.stderr.write(f"ERROR: {exc}")
                                   ^
SyntaxError: invalid syntax
The command '/bin/sh -c cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py && rm -f get-pop.py' returned a non-zero code: 1
```

**Comment by BELKHIR** [comment]
Hi Alexi,

Could you please change this type in Dockerfile and try again :
    RUN cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py && rm -f get-pop.py
to 
    RUN cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && python get-pip.py && rm -f get-pip.py

Thank's

**Comment by alxsimon** [comment]
Still the same error, I think the issue is before this `rm -f get-pip.py` command anyway.

**Comment by alxsimon** [comment]
What version of python are you using?
If python3 you can directly install pip from repositories:
`apt install python3-pip`

**Comment by alxsimon** [comment]
no it did not finish correctly in fact
```
/bin/sh: 1: pip: not found
The command '/bin/sh -c pip install Cython==0.27.3' returned a non-zero code: 127
```

**Comment by BELKHIR** [comment]
As in this stackoverflox answer : 
You have to use an older version of get-pip.py (the last version of pip doesn't support Python 2 anymore)
RUN curl https://bootstrap.pypa.io/2.7/get-pip.py --output get-pip.py && python get-pip.py && rm -f get-pip.py

**Comment by alxsimon** [comment]
Yes fixed by commit cd7130d41712eae8f1a2df0759b498739df1a008
